src/Microsoft.Android.Sdk.Analysis/Tests/DNAA0001Tests.cs

```diff
@@ -17,12 +17,15 @@ public async Task DNAA0001DoesNotShow ()
 	[Test]
 	[TestCase ("JniHandleOwnership")]
 	[TestCase ("Android.Runtime.JniHandleOwnership")]
+	[TestCase ("global::Android.Runtime.JniHandleOwnership")]
+	[TestCase ("AR.JniHandleOwnership")]
 	public async Task DNAA0001DoesNotShowForExistingCode (string type)
```

Comment:
Can we also test:
* `System.IntPtr`
* `IntPtr`
* `nint`
* `global::System.IntPtr`